setup.py

```diff
@@ -41,7 +41,7 @@
     author_email = 'pwang@enthought.com',
     maintainer = 'ETS Developers',
     maintainer_email = 'enthought-dev@enthought.com',
-    url = 'http://code.enthought.com/projects/chaco',
+    url = 'https://github.com/enthought/chaco'
```

Comment:
For this one I would keep it the way it is since my understanding is that code.enthought.com is the home page of the project and github is where we host the documentation. I know code.e.com is old and bad looking, but I don't think we have officially decided to throw it out. @dpinte @rkern an opinion?